legend-engine-external-shared-format-model/src/main/java/org/finos/legend/engine/external/shared/format/model/ExternalFormatModelGenerationExtension.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Goldman Sachs
```

Comment:
Why have two different extensions - ModelGen and SchemaGen for each external format? If they are mandatory, shouldn't they be merged into ExternalFormatExtension?